### PR TITLE
LPS-187829 [Marketplace] Create new page definition for Create New App component

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

**Comment by claricebrasil** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layouts/06_create_new_app/page-definition.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	"pageElement": {
+		"pageElements": [
+		],
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layouts/06_create_new_app/page-definition.json:4]: Don't we need to put the client extension here?

**Review by joanagrego — CHANGES_REQUESTED** [review]

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layouts/06_create_new_app/page-definition.json (hunk 1 above)
Comment (claricebrasil) [reply modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layouts/06_create_new_app/page-definition.json:4]: Fixed

**Comment by claricebrasil** [comment]
ci:test:sf

**Comment by claricebrasil** [comment]
ci:forward

**Comment by claricebrasil** [comment]
ci:forward:stop

**Comment by claricebrasil** [comment]
ci:forward:abort

**Comment by claricebrasil** [comment]
ci:forward:stop

**Comment by claricebrasil** [comment]
ci:stop

**Comment by claricebrasil** [comment]
ci:test:sf

**Comment by claricebrasil** [comment]
ci:test:sf

**Comment by claricebrasil** [comment]
ci:forward

**Comment by claricebrasil** [comment]
ci:test:stable

**Comment by joanagrego** [comment]
ci:forward

**Comment by claricebrasil** [comment]
ci:forward

**Pull request closed without merging**